x-pack/plugins/data_enhanced/public/search/sessions_mgmt/components/table/table.tsx

```diff
@@ -71,7 +72,8 @@ export function SearchSessionsMgmtTable({ core, api, timezone, config, ...props
   // initial data load
   useEffect(() => {
     doRefresh();
-  }, [doRefresh]);
+    plugins.data.search.usageCollector?.trackSessionsListLoaded();
```

Comment:
I think we might be able to replace this by using the sub view collector for the management app. Not sure we want to tho :thinking:  https://github.com/elastic/kibana/blob/master/src/plugins/kibana_usage_collection/server/collectors/application_usage/README.md